src/gebt_poc/section.h

```diff
@@ -20,19 +23,24 @@ class StiffnessMatrix {
 /// Class to manage normalized location, mass matrix, and stiffness matrix of a beam section
 class Section {
 public:
-    /// Constructor that initializes section with given location, mass, and stiffness
-    Section(double location, gen_alpha_solver::MassMatrix, StiffnessMatrix);
+    Section() : name_(""), location_(0.), mass_matrix_(), stiffness_matrix_() {}
+
+    Section(std::string name, double location, gen_alpha_solver::MassMatrix, StiffnessMatrix);
+
+    /// Returns the name of the section
+    inline std::string GetName() const { return name_; }
 
     /// Returns the normalized location of the section
     inline double GetNormalizedLocation() const { return location_; }
 
     /// Returns the mass matrix of the section
-    inline const gen_alpha_solver::MassMatrix& GetMass() const { return mass_matrix_; }
+    inline const gen_alpha_solver::MassMatrix& GetMassMatrix() const { return mass_matrix_; }
 
     /// Returns the stiffness matrix of the section
     inline const StiffnessMatrix& GetStiffnessMatrix() const { return stiffness_matrix_; }
 
 private:
+    std::string name_;                          //< Name of the section
```

Comment:
Why do sections need a name?

Reply:
It's a typical feature in FEA codes, imo. We might have different sections that are assigned to different elements of the model and it's a quick check to assert we have the right material/section assigned by checking the name.